src/app/action-items/action-items.component.html

```diff
@@ -1,38 +1,42 @@
-<div *ngIf="loading" class="spinner">
+<div *ngIf="isloadingActionItems" class="spinner">
     <i class="fa fa-spinner fa-spin"></i>
 </div>
-<div *ngIf="!loading" class="table-container" role="table" aria-label="Tasks">
+<div *ngIf="!isloadingActionItems" class="tableContainer" role="table" aria-label="Tasks">
     <div class="options">
-        <div class="assigned-label">
-            <div class="filter filter-opt">
-                <p class="filter-list">filters <i class="arrow fa fa-sort-down"></i></p>
+        <div class="assignedLabel">
+            <div class="filter filterOpt">
+                <p class="filterList">filters <i class="arrow fa fa-sort-down"></i></p>
             </div>
             <button class="filter assigned">Assigned to me X</button>
         </div>
-        <div class="new-item-button">
-            <button class="add-item">+ new action item</button>
+        <div class="newItemButton">
+            <button class="addItem" (click)="openDialog()">+ create action item</button>
         </div>
     </div>
-    <div class="flex-table header" role="rowgroup">
-        <div class="flex-row body-header first" role="columnheader">Title</div>
-        <div class="flex-row body-header" role="columnheader">Type</div>
-        <div class="flex-row body-header" role="columnheader">Completed</div>
-        <div class="flex-row body-header" role="columnheader">Due Date</div>
-        <div class="flex-row body-header" role="columnheader">Days Left</div>
+    <div class="tHead">
+        <div class="flexTable header" role="rowgroup">
+            <div class="flexRow bodyHeader first" role="columnheader">Title</div>
+            <div class="flexRow bodyHeader" role="columnheader">Type</div>
+            <div class="flexRow bodyHeader" role="columnheader">Completed</div>
+            <div class="flexRow bodyHeader" role="columnheader">Due Date</div>
+            <div class="flexRow bodyHeader" role="columnheader">Days Left</div>
+        </div>
     </div>
-    <div class="flex-table row" role="rowgroup" *ngFor="let t of dataSource">
-        <div class="flex-row body-cell first" role="cell">
-            <div class="title-container" id="circle-container">
-                <i class="fa fa-circle" [ngClass]="t.dueDay | circleColor"></i>
-            </div>
-            <div class="title-container">
-                <p class="action-item-name title">{{ t.title }}</p>
-                <p class="action-item-name project-name">{{ t.projectName }}</p>
+    <div class="tBody">
```

Comment:
**FIX:** Please provide UI tests validating if the action items data is displayed properly.